libs/community/langchain_community/llms/bedrock.py

```diff
@@ -93,11 +107,26 @@ class LLMInputOutputAdapter:
 
     @classmethod
     def prepare_input(
-        cls, provider: str, prompt: str, model_kwargs: Dict[str, Any]
+        cls,
+        provider: str,
+        model_kwargs: Dict[str, Any],
+        prompt: Optional[str] = None,
+        system: Optional[str] = None,
+        messages: Optional[List[Dict]] = None,
     ) -> Dict[str, Any]:
         input_body = {**model_kwargs}
         if provider == "anthropic":
-            input_body["prompt"] = _human_assistant_format(prompt)
+            if messages:
+                input_body["anthropic_version"] = "bedrock-2023-05-31"
```

Comment:
```suggestion
                    if "anthropic_version" not in input_body:
                        input_body["anthropic_version"] = "bedrock-2023-05-31"
```

Reply:
@Barneyjm 
One concern with this is that allowing another version would have to match the implementation of that version as well, in case it differs from what is currently here.

Reply:
yeah i agree it's not ideal, but providing some capability to pass this in will protect us in the future for whenever a new "anthropic_version" shows up.